script.js

```diff
@@ -8,4 +8,233 @@ bars.addEventListener('click', () => {
 
 list.addEventListener('click', () => {
   headers.classList.toggle('active');
-});
\ No newline at end of file
+});
+
+// creating a div element
+function iDiv(f = 'div') {
+  if (!f) {
+    f = 'div';
+  }
+  return document.createElement(f);
+}
+
+// project data declaration
+
+const myProjectData = [
+  {
+    id: 'port1',
+    title: 'Tonic',
+    framework: ['CANOPY', 'Back End Dev', 2015],
+    text: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    tags: ['HTML', 'CSS', 'Javascript'],
+    img: './images/snapshoot-portfolio1.png',
+    Details: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.',
+    liveLink: 'https://dmambo.github.io/',
+    sourceLink: '#',
+  },
```

Comment:
- [ ] kindly fix these design issues to match the Figma design : 

1. Kindly reduce the image size for the desktop popup to match the Figma design (mobile is fine)
2. kindly add the margin-bottom for the popup for both mobile and desktop versions as shown in Figma
3. your popup description should be aligned to the left as shown in Figma kindly fix it

**_figma design_**

![image](https://user-images.githubusercontent.com/89282221/202431424-70a4e9c9-b594-4a12-a683-6abc29293349.png)